app/controllers/cards_controller.rb

```diff
@@ -1,5 +1,69 @@
 class CardsController < ApplicationController
+  before_action :set_card, only: [:show, :edit, :update, :destroy]
+
   def index
     @cards = Card.all
+
+    respond_to do |format|
+      format.html
+    end
   end
+
+  def show
+    respond_to do |format|
+      format.html
+    end
+  end
+
+  def new
+    @card = Card.new
+
+    respond_to do |format|
+      format.html
+    end
+  end
+
+
+  def edit
+  end
+
+  def create
+    @card = Card.new(card_params)
+
+    respond_to do |format|
+      if @card.save
+        format.html { redirect_to @card, notice: t('card.created')  }
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.